lib/src/components/NxFilterDropdown/NxFilterDropdown.scss

```diff
@@ -40,12 +40,15 @@
   }
 
   &__reset {
+    color: var(--nx-color-link);
```

Comment:
Similar to the changes in `NxDropdown.scss`, I'm not understanding what this is for.  I'm not noticing any behavioral difference between your branch and main on this element.

Reply:
> Similar to the changes in NxDropdown.scss, I'm not understanding what this is for. I'm not noticing any behavioral difference between your branch and main on this element.

Oh I think I see.  It's because you changed this element to be an `.nx-dropdown-button`.